application/modules/invoices/views/modal_create_credit.php

```diff
@@ -62,7 +62,7 @@ function (data) {
                 <select name="invoice_group_id" id="invoice_group_id" class="hidden">
                     <?php foreach ($invoice_groups as $invoice_group) { ?>
                         <option value="<?php echo $invoice_group->invoice_group_id; ?>"
-                            <?php if (get_setting('default_invoice_group') == $invoice_group->invoice_group_id) {
+                            <?php if (get_setting('default_creditnote_group') == $invoice_group->invoice_group_id) {
```

Comment:
As requested in FR #763 (separate template and CNI number) and also because the current procedure in my country is not correct, I have created this PR. 
If I have to apply your proposal, invoice numbers and Credit Note numbers are mixed/used. 
See screenshot.
![image](https://github.com/user-attachments/assets/137a0f13-1d59-47f3-8e72-0c893f8ea0bb)
![image](https://github.com/user-attachments/assets/5e088bf8-0774-4ca9-8bde-176ac94a40bf)

I propose to implement a (credit note) choice or switch (**use invoice numbering or separate credit note numbering**) in "System Settings - Invoices" so that the user can decide for themselves which type they want to use.